src/performance/Processor.cpp

```diff
@@ -611,9 +611,9 @@ void Processor::pollAndProcessNewScores()
 	// Obtain all new scores since the last poll and process them
 	auto res = _pDBSlave->Query(StrFormat(
 		"SELECT `score_id`,`user_id`,`pp` "
-		"FROM `osu_scores{0}_high` "
-		"WHERE `score_id` > {1} ORDER BY `score_id` ASC LIMIT {3}",
-		GamemodeSuffix(_gamemode), _currentScoreId, _config.UserMetadataTableName, s_maxNumScores
+		"FROM `osu_scores{0}_high` JOIN score_process_queue USING (score_id) "
```

Comment:
I am not 100% sure whether backticks are necessary via the C mysql connector, but all existing queries here have them. You're missing them.